docs/blog/articles/2024-12-13-KServe-0.14-release.md

```diff
@@ -0,0 +1,149 @@
+
+You start by creating a node group as follows:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: LocalModelNodeGroup 
+metadata:
+  name: nodegroup1
+spec:
+  persistentVolumeSpec:
+    accessModes:
+    - ReadWriteOnce
+    volumeMode: Filesystem
+    capacity:
+      storage: 2Gi
+    hostPath:
+      path: /models
+      type: ""
+    persistentVolumeReclaimPolicy: Delete
+    storageClassName: standard
+  persistentVolumeClaimSpec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 2Gi
+    storageClassName: standard
+    volumeMode: Filesystem
+    volumeName: kserve
+
+```
+
+Then, you can specify to store an cache a model with the following resource:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterLocalModel
+metadata:
+  name: iris
+spec:
+  modelSize: 1Gi
+  nodeGroup: nodegroup1
+  sourceModelUri: gs://kfserving-examples/models/sklearn/1.0/model
+```
+
+<!--
+Related tickets:
+* Cluster local model controller: [#3860](https://github.com/kserve/kserve/pull/3860)
+* Cluster Local Model CR [#3839](https://github.com/kserve/kserve/pull/3839)
+* Add NodeDownloadPending status to ClusterLocalModel [#3955](https://github.com/kserve/kserve/pull/3955)
+-->
+
+## Support for Hugging Face hub in storage initializer
+
+The KServe storage initializer has been enhanced to support downloading models directly from Hugging Face. For this, the new schema `hf://` is now supported in the `storageUri` field of InferenceServices. The following YAML partial shows this:
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama3
+spec:
+  predictor:
+    model:
+      storageUri: hf://meta-llama/meta-llama-3-8b-instruct
+```
+
+Both public and private Hugging Face repositories are supported. The credentials can be provided by the usual mechanism of binding Secrets to ServiceAccounts, or by binding the credentials Secret as environment variables in the InferenceService.
+
+Read the [documentation](../../../modelserving/storage/huggingface/hf/) for more details.
+
+<!--
+Related tickets:
+* Implement Huggingface model download in storage initializer [#3584](https://github.com/kserve/kserve/pull/3584)
+-->
+
+## Other Changes
+
+This release also includes several enhancements and changes:
+
+### What's New?
+* New flag for automount serviceaccount token by [#3979](https://github.com/kserve/kserve/pull/3979)
+* TLS support for inference loggers [#3837](https://github.com/kserve/kserve/issues/3837)
+* Allow PVC storage to be mounted in ReadWrite mode via an annotation [#3687](https://github.com/kserve/kserve/issues/3687)
+
+### What's Changed?
+* Added `hostIPC` field to `ServingRuntime` CRD, for supporting more than one GPU in Serverless mode [#3791](https://github.com/kserve/kserve/issues/3791)
```

Comment:
I think it is good to add a section for LLM runtime support to include the changes as part of the 0.14 release
* vllm 0.6.x support
* add health endpoint for vLLM backend
* support shared memory volume for vLLM backend
* support chat completion template file
* support trust_remote_code for vllm and HF backend

Reply:
Do you mean a dedicated section with LLM-related enhancements? or should those be listed here under `What's Changed?`

Reply:
yes worth calling it out separately, cc @sivanantha321 to check if the list of changes are correct

Reply:
Ray is an optional dependency now and the way it is implemented is changed. It is worth mentioning this as a breaking change. https://github.com/kserve/kserve/pull/3834